apps/website/src/pages/docs/variables.mdx

```diff
@@ -27,56 +31,45 @@ import '@itwin/itwinui-variables';
 
 ## Variables
 
-### Background colors
+We strongly encourage developers to use iTwinUI's CSS variables rather than hardcoding values. Many of our color values change depending on the theme or contrast settings. Because of this, we purposefully do not expose the variables' values on this site. However, if you need the variables values you can extrapolate those values from [the raw CSS file](https://unpkg.com/browse/@itwin/itwinui-variables@2.x.x/index.css).
```

Comment:
I linked to https://unpkg.com/browse/@itwin/itwinui-variables@2.x.x/index.css on purpose incase we were to introduce iTwinUI-Variables 3.x and we wanted to version our documentation site.  But since we haven't introduced versioning to the doc site yet, maybe I'm thinking too far ahead.

Reply:
Linking to https://unpkg.com/@itwin/itwinui-variables/index.css now that this branch is targeting `dev`.

Reply:
Ok, using this link instead.